UMA-Bots/Funds-Deposited/src/findings.ts

```diff
@@ -0,0 +1,18 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export function createFinding(_metadata: {
+  amount: string;
+  originChainId: string;
+  destinationChainId: string;
+  token: string;
+}) {
+  return Finding.fromObject({
+    name: "SpokePool Funds Deposited Event Emitted",
+    description: `Deposited ${_metadata.amount} ${_metadata.token}`,
+    alertId: "UMA-1-1",
```

Comment:
```suggestion
    alertId: "UMA-1",
```
this is enough because there's no other findings